index.html

```diff
@@ -139,10 +155,12 @@ <h2 class="text-title">Uber Navigation</h2>
           <li>css</li>
           <li>Javascript</li>
         </ul>
-        <div class="but"><button class="button">See project</button></div>
+        <div class="but">
+          <a href="#works"><button class="button">See project</button></a>
+          </div>
       </div>
```

Comment:
- [ ] as requested by the previous reviewer  kindly consider removing the unused commented code it makes your code look messy and this will make your code base look clean